service/src/main/java/com/epam/rft/atsy/service/security/CustomSecurityContextHolderService.java

```diff
@@ -0,0 +1,31 @@
+package com.epam.rft.atsy.service.security;
+
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.stereotype.Service;
+
+/**
+ * Wrapper service for {@link SecurityContextHolder}.
+ * Used for isolating our dependency on Spring Security.
+ */
+@Service
+public class CustomSecurityContextHolderService {
```

Comment:
Please rename the class and also add an interface for the public methods.

eg: AuthenticationService for the interface, SpringSecurityAuthenticationService